src/lib/Value/ExportMethod.php

```diff
@@ -8,7 +8,7 @@
 
 namespace EzSystems\EzRecommendationClient\Value;
 
-class ExportMethod
+final class ExportMethod
 {
     const BASIC = 'basic';
```

Comment:
Those const's can be moved into their respective implementation classes. Then, considering that this class remains a public source of truth (or rather, can be one), you could import the values from the classes:
```php
public const BASIC = BasicMethodStrategy::EXPORT_TYPE;
```